Bottom: f92a4a392c31b6b1e249da16c44f06d492a4d932
Top:    f92a4a392c31b6b1e249da16c44f06d492a4d932
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 13:22:34 +0000

Use rtree of countours when computing an intersection

*** MORE COMPLEX, AND ACTUALLY SLOWS THING DOWN ***
(Might be faster if the r-tree already existed though)








---


